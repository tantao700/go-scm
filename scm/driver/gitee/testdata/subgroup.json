{
    "id": 14264161,
    "description": "Gitter Demo App",
    "name": "gitter-demo-app",
    "name_with_namespace": "gitee.org / gitter / gitter-demo-app",
    "path": "gitter-demo-app",
    "path_with_namespace": "gitee-org/gitter/gitter-demo-app",
    "created_at": "2019-09-11T23:30:02.188Z",
    "default_branch": "master",
    "tag_list": [],
    "ssh_url_to_repo": "git@gitee.com:gitee-org/gitter/gitter-demo-app.git",
    "http_url_to_repo": "https://gitee.com/gitee-org/gitter/gitter-demo-app.git",
    "web_url": "https://gitee.com/gitee-org/gitter/gitter-demo-app",
    "readme_url": "https://gitee.com/gitee-org/gitter/gitter-demo-app/blob/master/README.md",
    "avatar_url": null,
    "star_count": 0,
    "forks_count": 1,
    "last_activity_at": "2020-01-09T10:11:44.250Z",
    "namespace": {
        "id": 1540914,
        "name": "gitter",
        "path": "gitter",
        "kind": "group",
        "full_path": "gitee-org/gitter",
        "parent_id": 9970,
        "avatar_url": "/uploads/-/system/group/avatar/1540914/icon_128x128.png",
        "web_url": "https://gitee.com/groups/gitee-org/gitter"
    }
}