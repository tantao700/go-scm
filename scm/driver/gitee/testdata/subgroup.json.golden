{
    "ID": "14264161",
    "Namespace": "gitee-org/gitter",
    "Name": "gitter-demo-app",
    "Perm": {
        "Pull": true,
        "Push": false,
        "Admin": false
    },
    "Branch": "master",
    "Private": false,
    "Clone": "https://gitee.com/gitee-org/gitter/gitter-demo-app.git",
    "CloneSSH": "git@gitee.com:gitee-org/gitter/gitter-demo-app.git",
    "Link": "https://gitee.com/gitee-org/gitter/gitter-demo-app",
    "Created": "0001-01-01T00:00:00Z",
    "Updated": "0001-01-01T00:00:00Z"
}